app/test_split_integer.py

```diff
@@ -2,20 +2,22 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    assert sum(split_integer(17, 4)) == 17
 
 
 def test_should_split_into_equal_parts_when_value_divisible_by_parts() -> None:
-    pass
+    result_list = split_integer(16, 4)
+    for i in range(len(result_list) - 1):
+        assert result_list[i] == result_list[i + 1]
 
 
 def test_should_return_part_equals_to_value_when_split_into_one_part() -> None:
-    pass
+    assert split_integer(8,1 )[0] == 8
```

Comment:
There is an unnecessary space before the closing bracket in `split_integer(8,1 )`. It should be `split_integer(8, 1)` for consistency and readability.